tasks/session_management_service.yml

```diff
@@ -46,9 +46,10 @@
       PortalBackend__BaseUrl: "{{ admin_configuration.protocol }}://{{ admin_hostname }}"
       PortalBackend__ClientId: "{{ session_management_oauth2_client_id }}"
       PortalBackend__ClientSecret: "{{ session_management_oauth2_client_secret }}"
+      IpLookupService__ApiToken: "{{ session_management_configuration.ip_stack_api_token }}"
       CloudXRManagement__BaseUrl: "http://{{ container_names.cloudxr_management }}:5000"
       Sentry__Dsn: "{{ session_management_configuration.sentry_dsn }}"
-      CorsOrigins: "{{ session_management_configuration.CorsOrigins | default(admin_configuration.protocol + '://' + portal_hostname, true) }}"
+      CorsOrigins: "{{ session_management_configuration.cors_origins | default(admin_configuration.protocol + '://' + portal_hostname, true) }}"
```

Comment:
sneaked a little fix in there, didn't you? 😉

Reply:
I hoped you wouldn't see it 😉